src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorHelpers.cs

```diff
@@ -473,7 +473,8 @@ int sgn(T val)
 			// A new 'ArrayList' class is used to avoid an extra copy using List<T>.ToArray() here.
 			using (Stream stream = bitmap.PixelBuffer.AsStream())
 			{
-				_ = stream.WriteAsync(bgraPixelData.Array, 0, bgraPixelData.Count);
+				stream.Write(bgraPixelData.Array, 0, bgraPixelData.Count);
+				stream.Flush();
```

Comment:
I would be surprised if this is necessary. The using should automatically dispose/close the stream which automatically flushes it. If Flush() is needed I think it's hiding a bug in Uno.

Reply:
Flush might be unnecessary here. I just thrown all the extra measurements in during testing. I would revalidate this.

Reply:
Indeed, we don't need that `Flush()` here.